doc/user/prior-config-files.rst

```diff
@@ -0,0 +1,22 @@
+..
+.. January 12 2024, Christian Hopps <chopps@labn.net>
+..
+.. Copyright (c) 2024, LabN Consulting, L.L.C.
+..
+..
+
+Prior versions of FRR supported reading and writing per-daemon config files;
+however, with the introduction of the centralized management daemon ``mgmtd``
+this could no longer be supported.
+
+In order to allow for an orderly transition from per-daemon config files to the
+integrated config file, FRR daemons will continue to try and **read** their
+specific per-daemon configuration file as before. Additionally the config can
+still be loaded directly using the ``-f`` or ``--config-file`` CLI options;
+however, these files will **not** be updated when the configuration is written
+(e.g., with the ``write mem`` command).
+
+.. warning:: Per-daemon files will **no longer** be updated when the user issues
```

Comment:
I think it's required to be like:

```
.. warning:: 

   Per-daemon files will ...
```

Reply:
It renders the same both ways.

<img width="426" alt="image" src="https://github.com/FRRouting/frr/assets/3074665/ec184c3b-55d6-4089-9013-b3954e2bbf12">

Reply:
Good to know 👍